.github/workflows/build.yml

```diff
@@ -38,12 +38,12 @@ jobs:
   style_lint:
     if: github.repository == 'leanprover-community/mathlib'
     name: Lint style
-    runs-on: pr
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
       - name: install Python
-        if: ${{ ! 1 }}
+        if: ${{ 'ubuntu-latest' == 'ubuntu-latest' }}
```

Comment:
I don't know the syntax, but this looks like it will be always true.

Reply:
Yeah, these files are auto-generated from `build.yml.in` with `mk_build_yml.sh` so I used [some trickery](https://github.com/leanprover-community/mathlib/pull/9198/files#diff-4be11638e5a5a377ebd0a095300eea70486272a7fae5f5e2a4935d7f47a6bf23R24) to unify the logic here.